lib/zenodo/acts_as_zenodo_depositable.rb

```diff
@@ -43,6 +43,7 @@ def export_to_zenodo(access_token, extra_metadata = {})
         metadata.merge!(extra_metadata.to_h.deep_symbolize_keys)
 
         #FIXME: this is a quick hack
+        metadata[:title] = 'not set' if metadata[:title].blank?
```

Comment:
The Zenodo export needs thinking about, because the snapshot title/description may not be what you want to be the title/description of the Zenodo entry (currently it would be the title/description of the resource being snapshotted).